packages/flutter/lib/src/widgets/scroll_simulation.dart

```diff
@@ -197,7 +197,7 @@ class ClampingScrollSimulation extends Simulation {
     final double l = _splineDeceleration(velocity);
     final double decelMinusOne = _kDecelerationRate - 1.0;
     return friction *
-        _decelerationForFriction(0.84) *
+        _decelerationForFriction(0.779) *
```

Comment:
Is there any explanation why this needs to change from 0.84 to 0.779?

Reply:
This was the magic number to allow current golden tests to pass. To be honest I'm not sure why the tuning value must be different for Flutter. It could be that the incoming velocity is not pixels per second as expected by the Android logic or that the PPI assumption is invalid.